### ISSUE TITLE
Release for final circe

### ISSUE THREAD

**Issue by nafg** [tminglei/slick-pg#463]
For some reason 0.18.0 was released for a milestone release of Circe (0.12.0-M3). Can we get a release that uses a stable version of Circe (latest is 0.12.3)?

**Comment by tminglei** [comment]
Ok, I'll release a version for it.

**Comment by tminglei** [comment]
Well, looks not good
```
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 	::          UNRESOLVED DEPENDENCIES         ::
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 	:: io.circe#circe-core_2.11;0.12.3: not found
[warn] 	:: io.circe#circe-generic_2.11;0.12.3: not found
[warn] 	:: io.circe#circe-parser_2.11;0.12.3: not found
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
```

**Comment by tminglei** [comment]
Resolved and released `v0.18.1`.

**Comment by nafg** [comment]
> Well, looks not good
> 
> ```
> [warn] 	::::::::::::::::::::::::::::::::::::::::::::::
> [warn] 	::          UNRESOLVED DEPENDENCIES         ::
> [warn] 	::::::::::::::::::::::::::::::::::::::::::::::
> [warn] 	:: io.circe#circe-core_2.11;0.12.3: not found
> [warn] 	:: io.circe#circe-generic_2.11;0.12.3: not found
> [warn] 	:: io.circe#circe-parser_2.11;0.12.3: not found
> [warn] 	::::::::::::::::::::::::::::::::::::::::::::::
> ```

Yeah they dropped support for 2.11, like most libraries did when they added support for 2.13.

I'm sorry this caught you by surprise, in the future it's better not to make a stable release with unstable dependencies.
